### PR TITLE
Adding conribution instructions for quickstart workflow

### PR DESCRIPTION
**Pull request by ASHIQUEMD**
# Description

Adding instructions for contribution to Quickstart github workflow.

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #815

### CODE REVIEW

**Review by pravinpushkar — APPROVED** [review]
lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -107,6 +107,62 @@ git commit --amend --no-edit --signoff
...
+To contribute to the GitHub workflows in this repository, please follow these steps:
+
+1. Identify a specific area where you would like to contribute, such as adding a new workflow or improving an existing one.
+2. Fork the repository.
+3. Clone the repository to your local machine.
+4. Create a new branch for your changes: `git checkout -b <branch-name>`
+5. Make your changes to the GitHub workflows.
+6. Test your changes to ensure that they work as expected.
+7. Document any new workflows that you add or changes that you make to existing ones.
+8. Commit your changes: `git commit -am 'Add some feature'`
```
Comment (paulyuk) [inline CONTRIBUTING.md:121]: ideally -ams so its signed

**Review by paulyuk — APPROVED** [review]
LGTM.  Badly needed, thank you!

**Review by paulyuk — APPROVED** [review]
LGTM

**Pull request merged**